tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
```

Comment:
This is a different tense from the English ('sent' compared to 'sendas'). Assuming this string is used as, or just after, a user has sent out a monster, this looks like it might be a more natural translation than using 'sendis'.
Just wanted to bring this up in case it was accidental.